packages/ketcher-react/src/script/editor/tool/paste.ts

```diff
@@ -112,4 +227,31 @@ class PasteTool {
   }
 }
 
+type Template = {
+  aid?: number
+  molecule?: Struct
+  xy0?: Vec2
+  angle0?: number
+}
+function prepareTemplateFromSingleGroup(molecule: Struct): Template | null {
```

Comment:
May be we need to add some comment that this function calculates the position of FG and its rotation angle?

Reply:
Added.